src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const robot1 = {};
```

Comment:
what if you had another robot? would you name it robot2? try to describe what variable is having as a value. `newRobot` would be a great name